hydra_provider/proto/command_service.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package command_service;
+
+service CommandDispatcher {
+	rpc dispatch(Command) returns(Empty);
+}
+
+// NOTE: protobuf3 doesn't support inheritance for structuring commands better
+// SHOULD DO: find a more "composition" way of defining CommandData
+message Command {
+	CommandName name = 1;
+	CommandData data = 2;
+}
```

Comment:
ps:  you could technically just have a response like

```
message PowerDownResponse {
     int32 lastReportedVoltage = 2;
} 
```

but I'm assuming that anything can fail and if it does we return back the error. 

If something fails in rust, instead of doing an `unwrap()` we would return the error back to the sender with that `ErrorResponse` that you could pass a reason too.